_episodes/09-create.md

```diff
@@ -61,7 +61,7 @@ but most provide the following:
 
 Most databases also support Booleans and date/time values;
 SQLite uses the integers 0 and 1 for the former,
-and represents the latter as discussed [earlier](#a:dates).
+and represents the latter as discussed [earlier]({{ site.github.url }}/03-filter/).
```

Comment:
Should we link to the anchor `#date-types`?

Reply:
Where does `#date-types` resolve to. I can't seem to find anything related to #date-types

Reply:
https://swcarpentry.github.io/sql-novice-survey/03-filter/index.html#date-types